### PR TITLE
[NEW] Server Info page

### PR DESCRIPTION
**Pull request by gabriellsh**
## Screenshots
image

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] Improvement (non-breaking change which improves a current function)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Hotfix (a major bugfix that has to be merged asap)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ggazzo — APPROVED** [review]

File: client/views/admin/info/DeploymentCard.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+					{s(() => `${ statistics.migration.version } (${ formatDateAndTime(statistics.migration.lockedAt) }`)}
+				</Card.Col.Section>
+				<Card.Col.Section>
+					<Card.Col.Title>{t('MongoDB')}</Card.Col.Title>
+					{s(() => `${ statistics.mongoVersion } / ${ statistics.mongoStorageEngine } (oplog ${ statistics.oplogEnabled ? t('Enabled') : t('Disabled') })`)}
+				</Card.Col.Section>
+				<Card.Col.Section>
+					<Card.Col.Title>{t('Commit_details')}</Card.Col.Title>
+					{t('HEAD')}: ({s(() => commit.hash.slice(0, 9))}) <br />
+					{t('Branch')}: {s(() => commit.branch)}
```
Comment (jdauphant) [inline client/views/admin/info/DeploymentCard.js:55]: Hey @ggazzo , apparently if the app is deployed with a tar.gz (As it's done on multiple PaaS) :  git metadata aren't available (commit.hash and commit.branch). I describe the problem here : #21794

Comment (jdauphant) [reply client/views/admin/info/DeploymentCard.js:55]: Sorry ggazzo. It's @gabriellsh that I wanted to ping.

Comment (me-sp) [reply client/views/admin/info/DeploymentCard.js:55]: Sorry to ping you @gabriellsh, but is this problem #21794 fixed in newer versions (e.g. 3.15.1)?

Comment (me-sp) [reply client/views/admin/info/DeploymentCard.js:55]: We upgraded to version 3.16.1. (deployed with a tar.gz) the admin view info is working

**Pull request merged**